.github/scripts/simple-smoke-test.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+
+export CLV_VERSION=v5.2.0-beta3-preview1
+
+function test_docker_image() {
+    local image=$1
+    local container_name=$2
+    echo "Starting container '$container_name' from image '$image'"
+    docker run -it --name "$container_name" -d -p5701:5701 "$image"
```

Comment:
I am not sure where this is stored but maybe add `--rm` so the container is removed after being stopped.

Reply:
I've skipped '--rm' to be able to get logs and store them as artifacts. GitHub runners are cleared before each reuse